### ISSUE TITLE
Can we discuss how to break the model into separate component? I want to output the validation loss graph.

### ISSUE THREAD

**Issue by MheadHero** [sovit-123/fasterrcnn-pytorch-training-pipeline#19]
Hi sovit, its been a long time. I finally released from my finals. Now that I would like to continue to study this model, so that it could output the validation loss graph (because my thesis need it to check if the model overfit or underfit when applying it to an unseen dataset).

So, as last time, I had found a website: [How can I determine validation loss for faster RCNN (PyTorch)?](https://stackoverflow.com/questions/71288513/how-can-i-determine-validation-loss-for-faster-rcnn-pytorch)

And you have said it might required to break the model to several different module. So can you provide some guidance like how to break the model or how to implement the method used in the website?
